[th2-2334] prepare special build (#30)
th2-net/th2-conn-generic: conn-soup/README.md

> # Keeps all logical message parts together. If it set to value of `true` they won't be split to different batches

Reviewer: ```suggestion
# Maintains all the logical message parts together. If it is set to `true`, they won't be split into different batches
```

evolutionSupportEnabled: true
```

## Release notes

> ### 3.6.0

Reviewer 2: I think we should add information about the new base image version we've started using for the conn boxes

Author: I will add it to the main README file, thanks

> + Embedded service based on MINA decodes message as sender during send.

Reviewer: ```suggestion
+ Embedded service based on MINA decodes the message as sender during sending.
```

>   This approach is important for cases where pair of messages have the same protocol message type and different structures depending on direction.

Reviewer: ```suggestion
  This approach is important for cases in which a pair of messages have the same protocol message type and different structures depending on the direction.
```
